src/VirtualList/VirtualList.js

```diff
@@ -7,22 +7,36 @@
 import DOMUtil from '../Util/DOMUtil';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import throttle from 'lodash.throttle';
+
+import BindMixin from '../Mixin/BindMixin';
 import Util from '../Util/Util';
 
 let mathMax = Math.max;
```

Comment:
Not related to this PR but why are we _"caching"_ those utils? This saves basically nothing and is is esp. silly as we call `getVirtualState` in our on scroll handler which triggers layout/reflow (`getBoundingClientRect`, _`clientHeight`_).